.pre-commit-config.yaml

```diff
@@ -15,8 +15,7 @@ repos:
         files: "ml-agents-envs/.*"
         # Exclude protobuf files and don't follow them when imported
         exclude: ".*_pb2.py"
-        # TODO get disallow-incomplete-defs working
-        args: [--ignore-missing-imports, --follow-imports=silent]
+        args: [--ignore-missing-imports, --disallow-incomplete-defs]
```

Comment:
I think you still need `--follow-imports=silent` - when something imports protobuf code (even though it's ignored) that generates mypy errors (unless this option is on)

Reply:
Looks like this isn't needed, checks are passing 🤷‍♂

Reply:
Weird, sounds good though.